packages/common/src/lib/common/tools.ts

```diff
@@ -336,8 +353,8 @@ function getAppName(): string {
     return 'iobroker';
 }
 
+export const appNameLowerCase = 'iobroker';
```

Comment:
`getAppName` uses `isDevInstallation` which uses `lowercaseAppName` leading to a loop, hence I hardcoded it.